README.md

```diff
@@ -59,27 +66,34 @@ Download Constellation by going to the [release page](https://github.com/constel
 Unzip the constellation.zip bundle and double click the `bin/constellation64.exe` for Windows or
 run the `bin\constellation` shell script for Linux.
 
-There is currently no support for Mac and the feature request is tracked by [Issue #21](https://github.com/constellation-app/constellation/issues/21).
-
 ## Build Constellation
 
+* Constellation requires at least Open JDK 11 with JFX 11 support build to be 
+installed and is known to work on Windows 64-bit and Linux 64-bit.
+
 To build Constellation from source code do the following:
-[For Netbeans 11 with JDK 11 with JFX 11]
-* Download NetBeans 11 (http://netbeans.apache.org/download/nb112/nb112.html)
-* Download the Open JDK 11 with JFX 11 from Azul website for [Windows 64 bit build](https://cdn.azul.com/zulu/bin/zulu11.35.15-ca-fx-jre11.0.5-win_x64.zip) or [Linux 64 bit build](https://cdn.azul.com/zulu/bin/zulu11.35.15-ca-fx-jdk11.0.5-linux_x64.tar.gz)
-* Update the netbeans.conf file's netbeans_jdkhome entry (The file can be found under <C:\incubating-netbeans-11.0-bin\netbeans\etc> folder)
+
+* Download NetBeans 11.3 (https://netbeans.apache.org/download/nb113/nb113.html)
+* Download the Open JDK 11 with JFX 11 from Azul website for 
+[Windows 64 bit build](https://cdn.azul.com/zulu/bin/zulu11.37.19-ca-fx-jdk11.0.6-win_x64.zip) 
+or [Linux 64 bit build](https://cdn.azul.com/zulu/bin/zulu11.37.19-ca-fx-jdk11.0.6-linux_x64.tar.gz)
+* Update the netbeans.conf file's netbeans_jdkhome entry (The file can be found 
+under <C:\incubating-netbeans-11.0-bin\netbeans\etc> folder)
 * Clone this repository
-* Update the `netbeans_jdkhome` variable in `netbeans.conf`. Hint: This file may be located at `C:\Program Files\NetBeans 8.2\etc`.
+* Update the `netbeans_jdkhome` variable in `netbeans.conf`. Hint: This file may 
+be located at `C:\Program Files\NetBeans 8.2\etc`.
```

Comment:
Would it still be relevant to mention 8.2 here? 
Mine is located in C:\Program Files\NetBeans 11\Netbeans11\incubating-netbeans-11.0-bin\netbeans\etc